recipes/zoe/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import is_msvc_static_runtime
+from conan.tools.files import get, copy, apply_conandata_patches, export_conandata_patches
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.53.0"
+
+class ZoeConan(ConanFile):
+    name = "zoe"
+    description = "A multi-protocol, multi-threaded, resumable, cross-platform, open source, C++ file download library."
+    license = "GPL-3.0-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/winsoft666/zoe"
+    topics = ("curl", "download", "file", "ftp", "multithreading", "http", "libcurl", "rate-limit")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 11
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("libcurl/8.2.1")
```

Comment:
```suggestion
        self.requires("libcurl/[>=8.21 <9]")
```

We are accepting libcurl with version ranges already 😉  https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges